tests/python/pytorch/graphbolt/test_integration.py

```diff
@@ -1,10 +1,13 @@
 import dgl
 import dgl.graphbolt as gb
 import dgl.sparse as dglsp
+import pytest
 import torch
 
 
-def test_integration_link_prediction():
+@pytest.mark.parametrize("num_workers", [0])
```

Comment:
Why not set num_workers as [0, 4]?

Reply:
if `num_workers>0`, the return order of each iteration on dataloader is not determined.

Reply:
https://pytorch.org/docs/stable/notes/randomness.html#dataloader